<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGameDialog</class>
 <widget class="QDialog" name="AddGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="gameNameBox">
       <item>
        <widget class="QLabel" name="gameNameLabel">
         <property name="text">
          <string>Game name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="gameNameEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="gameDirBox">
       <item>
        <widget class="QPushButton" name="chooseGameDirButton">
         <property name="text">
          <string>Choose directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gameDirLabel">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>No game directory set</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="gameDirSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="gameExecutableBox">
       <item>
        <widget class="QPushButton" name="chooseGameExecutableButton">
         <property name="text">
          <string>Choose executable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gameExecutableLabel">
         <property name="text">
          <string>No executable set</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="gameExecutableSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="gameCommandLayout">
       <item>
        <widget class="QLabel" name="gameCommandLabel">
         <property name="text">
          <string>Open using:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="gameCommandEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="gameCommandBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="gameArgsLayout">
       <item>
        <widget class="QLabel" name="gameArgsLabel">
         <property name="text">
          <string>Arguments:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="gameArgsEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddGameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddGameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
